src/WindowsCaptureService/WindowsCaptureService.cpp

```diff
@@ -41,11 +45,30 @@ grpc::Status WindowsCaptureService::Capture(
   const CaptureOptions& capture_options =
       grpc_start_stop_capture_request_waiter.WaitForStartCaptureRequest();
 
+  if (capture_options.enable_api()) {
+    EnableApiInTracee(capture_options);
+  }
+
   StartEventProcessing(capture_options);
   TracingHandler tracing_handler{producer_event_processor_.get()};
   tracing_handler.Start(capture_options);
+
+  for (CaptureStartStopListener* listener : capture_start_stop_listeners_) {
+    listener->OnCaptureStartRequested(capture_options, producer_event_processor_.get());
+  }
+
   StopCaptureReason stop_capture_reason =
       grpc_start_stop_capture_request_waiter.WaitForStopCaptureRequest();
+
+  for (CaptureStartStopListener* listener : capture_start_stop_listeners_) {
+    listener->OnCaptureStopRequested();
+    ORBIT_LOG("CaptureStartStopListener stopped: one or more producers finished capturing");
```

Comment:
Is the logging intended here? Looks a bit like a left over from debugging ... the starting is not logged.

Reply:
Replicating the behavior on Linux, the start is not logged there either.

Reply:
ok, let's keep it as is.